package.json

```diff
@@ -27,6 +28,7 @@
     "gulp-watch": "^4.3.10",
     "http-server": "^0.9.0",
     "open-url": "^2.0.2",
+    "patch-package": "^6.4.7",
```

Comment:
avoid ^

Reply:
I think it should be ok to do this here. The `yarn.lock` file (roughly equivalent to package-lock.json for npm) should ensure the exact version is used https://classic.yarnpkg.com/lang/en/docs/yarn-lock/